rushScripts/regeneration.js

```diff
@@ -5,70 +5,75 @@ var exec = require('child_process').exec;
 swaggerDir = 'src/node_modules/@microsoft.azure/autorest.testserver/swagger/';
 
 goMappings = {
-    'additionalpropertiesgroup':['additionalProperties.json'],
-    'arraygroup':['body-array.json'],
-    'azurereportgroup':['azure-report.json'],
-    'booleangroup':['body-boolean.json'],
-    'bytegroup':['body-byte.json'],
-    'complexgroup':['body-complex.json'],
-    'custombaseurlgroup':['custom-baseUrl.json'],
-    'dategroup':['body-date.json'],
-    'datetimegroup':['body-datetime.json'],
-    'datetimerfc1123group':['body-datetime-rfc1123.json'],
-    'dictionarygroup':['body-dictionary.json'],
-    'durationgroup':['body-duration.json'],
-    'filegroup':['body-file.json'],
-    'formdatagroup':['body-formdata.json'],
-    'headergroup':['header.json'],
-    'httpinfrastructuregroup':['httpInfrastructure.json'],
-    'integergroup':['body-integer.json'],
-    'lrogroup':['lro.json'],
-    'modelflatteninggroup':['model-flattening.json'],
-    'morecustombaseurigroup':['custom-baseUrl-more-options.json'],
-    'numbergroup':['body-number.json'],
-    'optionalgroup':['required-optional.json'],
-    'paginggroup':['paging.json'],
-    'reportgroup':['report.json'],
-    'stringgroup':['body-string.json'],
-    'urlgroup':['url.json'],
-    'urlmultigroup':['url-multi-collectionFormat.json'],
-    'validationgroup':['validation.json'],
-    'xmlgroup':['xml-service.json'],
+    'additionalpropertiesgroup': 'additionalProperties.json',
```

Comment:
I'm thinking we should also comment this one out for now. We don't really have tests for it yet and it's not as stable. (We haven't gotten any errors because there isn't test coverage for it yet.)